lib/builderator/patch/thor-actions.rb

```diff
@@ -8,14 +7,10 @@ class Thor
   # Patch some Thor actions
   ##
   module Actions
-    RUN_OPTIONS = %w(cwd domain password user group umask timeout returns
-                     live_stream live_stdout live_stderr input logger log_level
-                     log_tag environment env login)
-
     ##
-    # Replace `run` with Mixlib::Shellout to accept STDIN
+    # Replace `run` with IO::popen to accept STDIN
```

Comment:
The `Thor::Actions.run` command doesn't accept `STDIN`. This implementation does. It's used for `packer`: https://github.com/rapid7/builderator/blob/debug-builders/lib/builderator/tasks/packer.rb#L25